src/server/graphql/mutations/setOrgUserRole.js

```diff
@@ -50,12 +53,12 @@ export default {
         })
         .count();
       if (leaderCount === 1) {
-        throw errorObj({_error: 'You’re the last leader, you can’t give that up'});
```

Comment:
So can we stop using this `errorObj` helper? I assume the GraphQL server runtime know how to capture regular `Error` objects with plain `message` fields and construct the appropriate response payload?

Reply:
yeah, graphql has gone through a few iterations on how they do errors & it's finally someplace good. so now i can do this formatting on the client.
In the future, for multi-field error messages, I think i'll add an `error` to the object type in the schema & just return it like normal. that way we don't have to do crafty things like serialize objects and call them messages.